pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -973,20 +972,17 @@ def refresh(self):
                 self._iso_chooser_button.set_use_underline(False)
         elif source_type == SOURCE_TYPE_HMC:
             self._hmc_button.set_active(True)
-        else:
-            # No method was given in advance, so now we need to make a sensible
-            # guess.  Go with autodetected media if that was provided, and then
-            # fall back to closest mirror.
+        elif source_type == SOURCE_TYPE_CDROM:
+            # Go with autodetected media if that was provided,
+            # otherwise fall back to the closest mirror.
             if not self._autodetect_button.get_no_show_all():
                 self._autodetect_button.set_active(True)
-                source_type = SOURCE_TYPE_CDROM
             else:
                 self._network_button.set_active(True)
```

Comment:
Should the network_button be active for SOURCE_TYPE_CDROM ?

Reply:
The option for CDROM ( `autodetect_button`) doesn't have to be visible if we are not able to find a valid installation media. In this case, we have to fall back to another option - the closest mirror (`network_button`).